tensorboard/plugins/hparams/hparams_demo.py

```diff
@@ -152,30 +179,31 @@ def run(logdir, session_id, hparams, group_name):
   # coefficient. But in real life, not everything is quite so clean, so
   # we'll add in some noise. (The value of 50 is arbitrary, chosen to
   # make the data look somewhat interesting. :-) )
-  noise = 50 * tf.random.normal([])
+  noise = 50 * tf.random_normal([])
   delta = -heat_coefficient * (ambient_difference + noise)
   scalar_summary.op('delta', delta,
                     description='The change in temperature from the previous '
                                 'step, in Kelvins.')
 
   # Collect all the scalars that we want to keep track of.
-  summ = tf.compat.v1.summary.merge_all()
+  summ = tf.summary.merge_all()
 
   # Now, augment the current temperature by this delta that we computed,
   # blocking the assignment on summary collection to avoid race conditions
   # and ensure that the summary always reports the pre-update value.
   with tf.control_dependencies([summ]):
     update_step = temperature.assign_add(delta)
 
-  sess = tf.compat.v1.Session()
-  sess.run(tf.compat.v1.global_variables_initializer())
-  for step in xrange(STEPS):
+  sess = tf.Session()
```

Comment:
Please revert the change and use compat.v1. Also, please test your code against tf-nightly and tf-nightly-2.0-preview

Reply:
Done.